WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailActivity.java

```diff
@@ -148,6 +148,14 @@ protected void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
     }
 
+    @Override
+    public void onBackPressed() {
```

Comment:
This for sure fixes the issue :)

The only thing that makes me thing a little bit more before merging this PR, is that we're clearing Notifications status bar from a details fragment. Note that `NotificationsDetailActivity` is also called from `WPMainActivity` when the app is started from PN https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java#L266.

Right before launching the Note details activity we're setting Notifications as default tab `mViewPager.setCurrentItem(WPMainTabAdapter.TAB_NOTIFS);` , but what if we will call this details activity from some other place in the future? Maybe without having `NotificationsList` in the back stack? Notifications will be cleared in the status bar, but the user isn't in the NotificationsList. 
Don't know if I was clear in the description...

I remember we had tackled a similar issue for Analytics in the past. See https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java#L328
Hope we can clear notifications there, only when the current active tab is `Notifications`. Would you mind to explore this other way?